wiki/Tournaments/MCNC/2019/en.md

```diff
@@ -285,51 +285,51 @@ The osu!mania 4K Chinese National Cup 2019 was run by various community members.
-5. The discord server will be the source of pretty much all the information for this tournament, so if you're joining, either as player, staff, or even as a spectator, you'll want to join for all of the latest updates. Players are required to join the official discord to be eligible of playing in the tournament. For those who cannot access discord, we have a QQ server containing the methods which could help you better access discord. Click here for the discord link.QQ server number: 597317576
+5. The discord server will be the source of pretty much all the information for this tournament, so if you're joining, either as player, staff, or even as a spectator, you'll want to join for all of the latest updates. Players are required to join the official discord to be eligible of playing in the tournament. For those who cannot access discord, we have a QQ server containing the methods which could help you better access discord. Click here for the discord link. QQ server number: 597317576
 6. For all of the details of the rules of the tourney, refer to the Information document here or in the discord channel. The discord channel and that document will have all the information you need, and the discord channel will be where future announcements will be made! So make sure you join the channel if you are participating.
 
-### Tournament Registration
+### Tournament registration
 
 1. There is no rank range for this tournament, but players who have cleared the 9th dan (96% accuracy in score v1) in the regular 4k Dan course is highly recommended to join. But please feel free to join if you are below this recommended level, as it's a great opportunity to improve and make new friends.
 2. To be eligible for playing, one must:
-    - Have more than 30 hours of playtime or 1,000,000 total hits in osu!mania.
-    - Have not been banned/majorly restricted in the past year.
-    - Pass a background check by osu!staff and the tournament staff.
-    - Include the answer "lfnb!" in the question "Do you have anything else to say?"
+   - Have more than 30 hours of playtime or 1,000,000 total hits in osu!mania.
+   - Have not been banned/majorly restricted in the past year.
+   - Pass a background check by osu!staff and the tournament staff.
+   - Include the answer "lfnb!" in the question "Do you have anything else to say?"
 3. We reserve the right to deny any and all suspicious accounts/players from participating in the tournament.
 Players must abide to the osu! terms of service.
```

Comment:
is this part of rule 3 above?

Reply:
this is part of rule 2

Reply:
he was asking about line 306, I'm going to combine 305 and 306